cirq-core/cirq/contrib/acquaintance/executor.py

```diff
@@ -100,6 +117,67 @@ def optimization_at(
         )
 
 
+class StrategyExecutorTransformer:
+    """Executes an acquaintance strategy."""
+
+    def __init__(self, execution_strategy: ExecutionStrategy) -> None:
+        """Initializes transformer.
+
+        Raises:
+            ValueError: if execution_strategy is None.
+        """
+
+        if execution_strategy is None:
+            raise ValueError('execution_strategy cannot be None')
+        self.execution_strategy = execution_strategy
+        self.mapping = execution_strategy.initial_mapping.copy()
+
+    def __call__(
+        self, circuit: circuits.Circuit, context: Optional['cirq.TransformerContext'] = None
+    ) -> circuits.Circuit:
+        """Executes an acquaintance strategy using primitive map function and
+          mutates initial mapping.
+
+        Args:
+            circuit: 'circuits.Circuit' input circuit to transform.
```

Comment:
```suggestion
            circuit: 'cirq.Circuit' input circuit to transform.
```

Reply:
always reference the base module for an import in docstring?

Reply:
Yes, the docstrings should always be `cirq.<object_name>` because in this format, the docs infrastructure automatically generates hyperlinks to reference pages of the objects on the website.

Reply:
Good to know. Thanks!